app/docs/0.10.x/plugin-development/entities-cache.md

```diff
@@ -73,7 +73,7 @@ Bringing back our authentication plugin example, to lookup a credential with a s
 local function load_entity_key(api_key)
   -- IMPORTANT: the callback is executed inside a lock, hence we cannot terminate
   -- a request here, we MUST always return.
-  local apikeys, err = dao.apikeys:find_by_keys({key = api_key}) -- Lookup in the datastore
+  local apikeys, err = dao.apikeys:find({key = api_key}) -- Lookup in the datastore
```

Comment:
The method that replaced `find_by_keys` is `find_all`. `find()` expects only the primary key of an entity and returns a single value, not an array of rows. Sorry!